main.py

```diff
@@ -0,0 +1,57 @@
+import time
+from turtle import Screen
+from paddle import Paddle
+from ball import Ball
+from scoreboard import Scoreboard
+
+direction = 10
```

Comment:
same comments as the last PR.  Lots of hard-coded values that I'd shift to the top of the file and, ideally, in a JSON.

Reply:
converted variables in constants and placed at top of each file.